### ISSUE TITLE
Code for visualizations 

### ISSUE THREAD

**Issue by krishnakanthnakka** [columbia/Metric_Learning_Adversarial_Robustness#1]
Can you please share the code for plotting visualizations (t-SNE plots) in the paper.  t-SNE plots have tunable parameters like perplexity, the number of iterations, learning rate, etc.

This will be helpful for reproducing results. Btw, did you use Sklearn's t-SNE with default parameters?

**Comment by ChengzhiCU** [comment]
We use multi-core t-SNE, https://github.com/DmitryUlyanov/Multicore-TSNE. We will upload the code shortly.

**Comment by AIasd** [comment]
We have added a script for it. You can check it out and reproduce our results by running it.
